go.mod

```diff
@@ -9,7 +9,6 @@ require (
 	github.com/Azure/go-ansiterm v0.0.0-20210617225240-d185dfc1b5a1 // indirect
 	github.com/Microsoft/go-winio v0.4.16 // indirect
 	github.com/StackExchange/wmi v0.0.0-20190523213315-cbe66965904d // indirect
-	github.com/axw/gocov v1.0.0 // indirect
```

Comment:
Please do not exclude gocov and gocov-html from go.mod.

Reply:
Done